src/sec_certs/sample/fips.py

```diff
@@ -629,6 +637,26 @@ def extract_policy_pdf_keywords(cert: FIPSCertificate) -> FIPSCertificate:
             cert.pdf_data.keywords = keywords
         return cert
 
+    @staticmethod
+    def extract_br1_metadata(cert: FIPSCertificate) -> FIPSCertificate:
+        """
+        Extract br1 chapters and tables from the document
+        """
+        with Path(cert.state.policy_txt_path).open() as f:
```

Comment:
When #525 lands, this should perhaps be gated behind checking that the PDFConverter is the Docling one. Because if it is not, it does not even make sense to run all of this, because it will not produce the proper chapters right?